app/views/runs/show.html.erb

```diff
@@ -44,10 +44,25 @@
           <td class="testaction-toggle run-test-name-column cursor-hand"><%= run_test.name %></td>
           <td class="testaction-toggle run-test-description-column"><%= run_test.description %></td>
           <td class="run-test-success-column"><%= display_test_status(run_test) %><div class="clearfix"></div></td>
+          <td class="carousel-playback-btn"><button class="btn btn-primary" id="hideshow" value="hide/show">Playback</button></td>
+
         </tr>
         <tr class="testaction-row">
-          <td colspan='3'>
+        <div class="outer-scroll" id="outer-scroll" style="display: none">
+        <div class="scroller" style="margin:0 auto;">
+                <%- for @screenshot in run_test.screenshots(run_browser_id(run_test.id)) %>
+                <%- for @base64string in @screenshot %>
+                    <div style="margin:0 auto;">
+                    <img src="data:image/png;base64,<%= @base64string %>" align= "center">
+                    </div>
+                <%- end %>
+                <%- end %>
```

Comment:
I think this block should be moved out of the view. Maybe something like this: `@screenshots_block = some_function` in the controller, where `some_function` is defined in the Runs model file, and called here (replace the whole thing with `<%= @screenshots_block %>`